homeassistant/helpers/selector.py

```diff
@@ -453,6 +453,27 @@ def __call__(self, data: Any) -> int:
         return value
 
 
+class ConditionSelectorConfig(TypedDict):
+    """Class to represent an action selector config."""
+
+
+@SELECTORS.register("condition")
+class ConditionSelector(Selector[ConditionSelectorConfig]):
+    """Selector of an condition sequence (script syntax)."""
+
+    selector_type = "condition"
+
+    CONFIG_SCHEMA = vol.Schema({})
```

Comment:
Should it use `vol.All(ensure_list, [CONDITION_SCHEMA])` ?

Reply:
The schema must be empty because there is no options for this selector.